root/opt/nethvoice-report/scripts/queue-miner.php

```diff
@@ -48,6 +48,19 @@
 # Delete queue_log
 $sqls[] = "DELETE FROM queue_log WHERE UNIX_TIMESTAMP(time) < ?";
 
+// Create queue_failed table if it doesn't exists
+$sqls[] = "
+    CREATE TABLE IF NOT EXISTS `queue_failed` (
+      `cid` varchar(100) NOT NULL DEFAULT '',
+      `name` varchar(80) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+      `company` varchar(80) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+      `action` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+      `time` INT(11) NOT NULL DEFAULT 0,
+      `direction` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+      `qname` varchar(32) NOT NULL DEFAULT '',
+      `event` varchar(20) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT ''
+    ) ENGINE=InnoDB AUTO_INCREMENT=398433 DEFAULT CHARSET=utf8";
+
```

Comment:
No way :)

Please move table creation to the right place: https://github.com/nethesis/nethvoice-report/blob/master/root/opt/nethvoice-report/scripts/schema.sql

Also *remove* the AUTO_INCREMENT option.